src/util/Utils.js

```diff
@@ -33,4 +35,35 @@ export default class Utils {
     }
     return [object_or_array];
   }
+
+  static findChildren(question, id, transitive, reflexive) {
```

Comment:
No sure what u are doing here. For given ID you can find only one question, so there is no sense in returning array. But anyway i don't understand why you are passing "id" parameter there. I did not use `id` parameter when explaining it here: https://github.com/kbss-cvut/s-forms/issues/172 ... On the other hand if you are using `id` parameter i do not understand then why you are using `JsonLdObjectUtils.checkId(subQuestions, startingQuestion)`.

I was expecting this method (did not tested):
```
static getSubQuestions(question, transitive, reflexive) {
    const subQuestions = asArray(question[Constants.HAS_SUBQUESTION]); // we have such method in json-ld utils 
    let questions = [];

    if (reflexive) {
        questions.push(question);
    }
    
    if (transitive) {
        for (q : subQuestions) {
          findChildren(question, true, true)
        }
    } else {
        question.push(subQuestions);
    }
    
    return questions;
  }
```

Reply:
```
 static findChild(parent, id) {

    if (!parent) {
      return null;
    }

    if (parent['@id'] === id) {
      return parent;
    }

    const subQuestions = parent[SConstants.HAS_SUBQUESTION];
    if (subQuestions && subQuestions.length) {

      for (let subQuestion of subQuestions) {
        let found = Utils.findChild(subQuestion, id);
        if (found) {
          return found;
        }
      }
    }

    return null;
  }
  ````